### PR TITLE
Added isDisplayed, scrollIntoView and hover functionality

### PR DESCRIPTION
**Pull request by ophirg**

### CODE REVIEW

**Comment by ophirg** [comment]
I will take care of improving the todo-app, as described https://github.com/wix-incubator/unidriver/pull/18#discussion_r244934363 in another branch

**Comment by GabiGrin** [comment]
@ophirg  thanks a lot for this! and sorry for the delay
as discussed, the isDisplayed in jsdom needs some thinking and the hardcoded `true` might be confusing. I

I'll release v2

File: package.json [hunk 1]
```diff
@@ -44,21 +42,23 @@
...
-    "puppeteer": "^0.13.0",
+    "puppeteer": "^1.11.0",
     "protractor": "^5.4.1",
     "react": "^15.2.0",
     "react-test-utils": "0.0.1",
     "react-dom": "^15.2.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "protractor": "^5.4.2",
```
Comment (shlomitc) [inline package.json]: @GabiGrin - this looks wrong. I’d expect it to be a peerDep

Comment (GabiGrin) [reply package.json]: true! missed that, will fix and release

**Pull request merged**